docs/configuring_tor.md

```diff
@@ -1,25 +1,4 @@
-### Table of Contents
-1. [Overview](#Overview)<br />
-2. [Client-Only](#Client)<br />
-2.1 [Description](#ClientDescription)<br />
-2.2 [Command Line Example](#ClientCLIExample)<br />
-2.3 [Config File Example](#ClientConfigFileExample)<br />
-3. [Client-Server via Tor Hidden Service](#HiddenService)<br />
-3.1 [Description](#HiddenServiceDescription)<br />
-3.2 [Command Line Example](#HiddenServiceCLIExample)<br />
-3.3 [Config File Example](#HiddenServiceConfigFileExample)<br />
-4. [Bridge Mode (Not Anonymous)](#Bridge)<br />
-4.1 [Description](#BridgeDescription)<br />
-4.2 [Command Line Example](#BridgeCLIExample)<br />
-4.3 [Config File Example](#BridgeConfigFileExample)<br />
-5. [Tor Stream Isolation](#TorStreamIsolation)<br />
-5.1 [Description](#TorStreamIsolationDescription)<br />
-5.2 [Command Line Example](#TorStreamIsolationCLIExample)<br />
-5.3 [Config File Example](#TorStreamIsolationFileExample)<br />
-
-<a name="Overview" />
-
-### 1. Overview
+# Configuring TOR
```

Comment:
Shouldn't this content belong to  `configuration.md`?

Reply:
i think tor is a complicated thing and the normal configuration.md should focus on the default configuration. i would like to suggest keeping it in an own file.